### PR TITLE
Update remove-former-employee.md

### PR DESCRIPTION
**Pull request by msbemba**
Correct navigation to Initiate Sign-out and added a note that only GA can do this.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2203

**Comment by msbemba** [comment]
@JohanFreelancer9  Please copy review

File: microsoft-365/admin/add-users/remove-former-employee.md [hunk 1]
```diff
@@ -79,10 +79,13 @@ To prevent an employee from logging in:
 
 3. Enter a new password, and then select **Reset**. (Don't send it to them.)
 
-4. Select the user again, expand **OneDrive Settings**, and then select **Initiate** next to **Sign-out**.
+4. Select the user's name to go to their properties pane, and on the **Account** tab, select **Initiate sign-out**.
 
 ::: moniker-end
 
+> [!NOTE]
+> You need to be a Global Administrator to Intiate Sign-out.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/remove-former-employee.md]:
```suggestion
> You need to be a global administrator to initiate sign-out.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KwekuA Please Merge this PR. Thanks.

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**